nf_core/pipeline-template/bin/scrape_software_versions.py

```diff
@@ -11,7 +11,7 @@
         software = "{{ name }}"
 
     with open(version_file) as fin:
-        version = fin.read().strip()
+        version = fin.read().strip().replace("\n", ", ")
```

Comment:
With the yaml strings the entire file can be deleted?

Reply:
I think so, but I wanted to check if it was needed for anything else I didn't know about.